assets/migrations/mysql/003_reverse_lookup_index.sql

```diff
@@ -0,0 +1,5 @@
+-- +goose Up
+CREATE INDEX idx_reverse_lookup_user on tuple (store, object_type, relation, _user);
```

Comment:
Maybe rename this file to `003_add_connected_objects_to_user_index.sql`

Reply:
The name of the file matches the index being added. Do you recommend we change the name of the index? I thought it made sense to have a reverse_lookup index since we're reverse looking up on the `user` field.

Reply:
Oh I kind of wanted to follow the pattern of the previous migrations that start with a verb: `initialize`, `add`, etc.

Reply:
Ok, I'll rename to `003_add_reverse_lookup_index.sql`.